dev-start-ui.sh

```diff
@@ -5,4 +5,12 @@ export GU_sns_errors="not-used"
 export GU_dbkey="gudocs-dev"
 export GU_require_domain_permissions="dev-guardian.co.uk"
 
+export NVM_DIR="$HOME/.nvm"
```

Comment:
This is quite similar to the dev-start-fetch script. Is there a reason they both do the nvm export and install?

Reply:
Good idea. I guess we can take it that someone has run the UI script before the fetch script so could probably take it out of here.